### PR TITLE
Upgraded UI package

### PR DESCRIPTION
**Pull request by idanstark42**

### CODE REVIEW

**Comment by 2roy999** [comment]
Hi, I created a commit, changing the karma settings so we can see the passed and failed tests on appveyor (I fixed my single comment review on the way).

**Comment by idanstark42** [comment]
The icon does not change when the window is opened, so some of the tests we're irrelevant.

File: test/client/settings.test.jsx [hunk 1]
```diff
@@ -16,67 +16,31 @@ const { mount } = Enzyme
...
 
   it('shows the sound off icon when created without sound-window key in the local storage', () => {
     const wrapper = mount(<Settings />)
 
-    expect(wrapper.find('.fas')).to.have.className('fa-volume-off')
+    expect(wrapper.find('i.icon')).to.have.className('music')
   })
 
   it('shows the sound up icon when created with sound-window key in the local storage', () => {
-    window.localStorage.setItem('sound-window', true)
```
Comment (2roy999) [inline test/client/settings.test.jsx]: Why did you delete all of these tests? they are checking logic that is still exists

Comment (idanstark42) [reply test/client/settings.test.jsx]: No... it doesn't. The icon doesn't change anymore. Why did we use a changing icon here in the first place? The very indication of the window being open is seeing it open, there's no need for indication on the button.

Comment (2roy999) [reply test/client/settings.test.jsx]: The change of the icon is not the important part, what's important is that you prevent a case of two of those windows opening.

Comment (idanstark42) [reply test/client/settings.test.jsx]: Ok so I changed it to only remove the tests:
 - changes the icon when sound-window appears in the local storage
 - changes the icon when sound-window removed from the local storage
